src/client/pfs.go

```diff
@@ -719,14 +719,24 @@ func (c APIClient) ListFile(repoName string, commitID string, path string) ([]*p
 	if err != nil {
 		return nil, grpcutil.ScrubGRPC(err)
 	}
-	return fileInfos.FileInfo, nil
+	result := make([]*pfs.FileInfo, 0, 10)
```

Comment:
Any particular reason with start with a capacity of 10 here?

Reply:
No. I decided it was pointless and got rid of it